projects/plugins/boost/app/lib/minify/functions-helpers.php

```diff
@@ -240,11 +181,9 @@ function jetpack_boost_page_optimize_schedule_cache_cleanup() {
 	$cache_folder = Config::get_cache_dir_path();
 	$args         = array( $cache_folder );
 
-	$cache_cleanup_hook = Config::get_cron_cache_cleanup_hook();
-
 	// If caching is on, and job isn't queued for current cache folder
-	if ( false !== $cache_folder && false === wp_next_scheduled( $cache_cleanup_hook, $args ) ) {
-		wp_schedule_event( time(), 'daily', $cache_cleanup_hook, $args );
+	if ( false !== $cache_folder && false === wp_next_scheduled( 'page_optimize_cron_cache_cleanup', $args ) ) {
+		wp_schedule_event( time(), 'daily', 'page_optimize_cron_cache_cleanup', $args );
```

Comment:
Shouldn't the event name be changed to boost?

Reply:
You're right. Updated in 75907dcc1b043ff092a49c4cb621d875fbf537c9.